common/changes/@itwin/core-backend/Soham-fixing_test_for_cte_with_alias_2025-01-07-07-35.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "Added and updated test cases for alias support in CTE subquery",
```

Change logs are for end users. End users don't care about tests.
```suggestion
      "comment": "",
```